src/components/JudgeInterface/JudgeInterface.tsx

```diff
@@ -66,21 +91,67 @@ export default function JudgeInterface(): JSX.Element {
   return (
     <div className="relative h-full flex flex-col">
       <div className="p-4 pb-0">
-        <p className="text-gray-100 font-bold text-lg">
-          {settings.judgeUrl}
-          {/* Problem 1. Social Distancing I */}
-        </p>
+        {problemData && problemData.parsed ? (
+          <>
+            <p className="text-gray-100 font-bold text-lg">
+              <a
+                href={getUSACOContestURL(problemData.source || '')}
+                className="text-indigo-300"
+                target="_blank"
+                rel="noreferrer"
+              >
+                {problemData?.source}
+              </a>
+            </p>
+            <p className="text-gray-100 font-bold text-lg">
+              <a
+                href={`http://www.usaco.org/index.php?page=viewproblem2&cpid=${problemID}`}
+                className="text-indigo-300"
+                target="_blank"
+                rel="noreferrer"
+              >
+                {problemData?.title}
+              </a>
+            </p>
+            <p className="text-gray-100 text-sm">
+              <span className="font-bold">INPUT:</span> {problemData?.input}
+            </p>
+            <p className="text-gray-100 text-sm">
+              <span className="font-bold">OUTPUT:</span> {problemData?.output}
+            </p>
+          </>
+        ) : (
+          <>
+            <p className="text-gray-100 font-bold text-lg">
+              USACO Problem ID:{' '}
+              <a
+                href={`http://www.usaco.org/index.php?page=viewproblem2&cpid=${problemID}`}
+                className="text-indigo-300"
+                target="_blank"
+                rel="noreferrer"
+              >
+                {problemID}
+              </a>
+            </p>
+            <p className="text-red-300 text-sm font-bold">
+              {problemData
+                ? 'Could not parse problem data.'
+                : 'Problem ID does not exist.'}
+            </p>
+          </>
+        )}
         <p className="text-gray-400 text-sm">
           Early access. Report issues to Github. Do not spam submit.
-          {/* USACO 2020 US Open Contest, Bronze */}
+          {/* Note: You will not be able to submit to a problem in an active contest. */}
+          {/* ^ is this necessary? */}
```

Comment:
idk I feel like some complete beginners might get confused and try to use this to submit :P